com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/page/view/SynonymMappingView.java

```diff
@@ -277,6 +277,10 @@ void setSynonym(Synonym synonym) {
 		 * @param editable
 		 */
 		void setEditable(boolean editable) {
+			if (config.targetIsOnline() 
+					&& Integer.parseInt(config.getTargetDBVersion()) < 112) {
```

Comment:
what about use of USERSCHEMA_VERSION instead of 112

Reply:
It's a small thing, but it seems important.
Thank you for letting me know.